site/docs/transform/window.md

```diff
@@ -0,0 +1,91 @@
+| percent_rank | _None_    | Assigns a percentage rank order value to each data object in a window. The percent is calculated as _(rank - 1) / (group_size - 1)_. |
+| cume_dist    | _None_    | Assigns a cumulative distribution value between 0 and 1 to each data object in a window.|
+| ntile        | Number | Assigns a quantile (e.g., percentile) value to each data object in a window. Accepts an integer parameter indicating the number of buckets to use (e.g., 100 for percentiles, 5 for quintiles).|
+| lag          | Number | Assigns a value from the data object that precedes the current object by a specified number of positions. If no such object exists, assigns `null`. Accepts an offset parameter (default `1`) that indicates the number of positions. This operation must have a corresponding entry in the _fields_ parameter array.|
+| lead         | Number | Assigns a value from the data object that follows the current object by a specified number of positions. If no such object exists, assigns `null`. Accepts an offset parameter (default `1`) that indicates the number of positions. This operation must have a corresponding entry in the _fields_ parameter array.|
+| first_value  | _None_    | Assigns a value from the first data object in the current sliding window frame. This operation must have a corresponding entry in the _fields_ parameter array.|
+| last_value   | _None_    | Assigns a value from the last data object in the current sliding window frame. This operation must have a corresponding entry in the _fields_ parameter array.|
+| nth_value    | Number | Assigns a value from the nth data object in the current sliding window frame. If no such object exists, assigns `null`. Requires a non-negative integer parameter that indicates the offset from the start of the window frame. This operation must have a corresponding entry in the _fields_ parameter array.|
+
+## Examples
+
+Below are some common use cases for the window transform. Shown are the examples along side the Vega Lite spec.
+
+### Percent of Total
+
+<div class="vl-example" data-name="window_transform_activities"></div>
+
+### Difference from Mean
+
+<div class="vl-example" data-name="window_transform_movie_mean_difference"></div>
+
+Another example is to show the best movies for the year they were released. Here best is defined by having a score that is 2.5 points higher than the average for the year it was released in.
+
+<div class="vl-example" data-name="window_transform_mean_difference_by_year"></div>
+
+Rather than filtering the above two examples we can also show a residual graph using the window transform.
+
+<div class="vl-example" data-name="window_transform_residual_graph"></div>
+
+To replace any of the examples from total to `mean`, the operation just needs to be changed from `sum` to `mean`.
```

Comment:
This sentence is appearing twice.

Also, I don't understand why this sentence is necessary here especially since residual graph already uses "mean". 

As mentioned earlier on slack, we better just remove it.